### DISCUSSION TITLE
[Unearth] Is the viewer styling static or is there a way to style it differently?

### DISCUSSION THREAD

**Discussion by gabe-unearth** [iTwin/itwinjs-core#2745] [Q&A] [answered]
[Unearth] Is the viewer styling static or is there a way to style it differently? (e.g. color scheme, chrome, etc.)

**Comment by aruniverse** [comment, marked as answer]
With the viewer we support two themes by default ( "dark" and "light" ), but you can override our default styles, but it'll take some work.

Here's a really dumb example, but shows it's possible: https://www.itwinjs.org/sandboxes/ArunGeorge/universal-cyan-elephant-265
In devtools, you can inspect and look for the classnames of the elements that you want to override.
